src/rpc/blockchain.cpp

```diff
@@ -1037,7 +1037,7 @@ static RPCHelpMan getblock()
                                             {RPCResult::Type::STR, "asm", "The asm"},
                                             {RPCResult::Type::STR, "hex", "The hex"},
                                             {RPCResult::Type::STR, "address", /* optional */ true, "The Bitcoin address (only if a well-defined address exists)"},
-                                            {RPCResult::Type::STR, "type", "The type, eg 'pubkeyhash'"},
+                                            {RPCResult::Type::STR, "type", "The type (" + GetAllOutputTypes() + ")"},
```

Comment:
nit

Existing other case says "(e.g. " ...

Maybe "(one of: " would be better.

Reply:
I agree that `one of:` sounds good if we are enumerating all possible options.

Let's also change `src/rpc/rawtransaction.cpp:568`

Reply:
Thank you. Modified.